### ISSUE TITLE
System Temperature should default to Celsius

### ISSUE THREAD

**Issue by HeedfulCrayon** [travisghansen/hass-pfsense#58]
I believe this is something that needs to be fixed within the integration according to this github issue
https://github.com/home-assistant/core/issues/61163

Basically, my system is set to imperial units, but for things like CPU temp, I want it in celsius

**Comment by travisghansen** [comment]
It is reported as celsius actually. It’s just the hass ui that displays it per your settings.

**Comment by HeedfulCrayon** [comment]
Did you read the github issue I linked?  There is no way for me to convert this to Celsius like you used to be able to do, and that is because of a change of native_unit_of_measurement.  This is something that has to be fixed at the integration level (if I understand the issue correctly).
I have tried customizing the sensor and setting the unit_of_measurement to Celsius, but that actually changes that number to a Celsius measurement and then converts it to Fahrenheit (which makes it terribly incorrect)
